clients/include/testing_asum.hpp

```diff
@@ -103,35 +88,31 @@ rocblas_status testing_asum(Arguments argus)
         CHECK_ROCBLAS_ERROR(rocblas_set_pointer_mode(handle, rocblas_pointer_mode_device));
         status = rocblas_asum<T1, T2>(handle, N, dx, incx, d_rocblas_result_2);
 
-        asum_arg_check(status, d_rocblas_result_2);
```

Comment:
We have to add <T2> explicit template argument, to force the second argument to T2* type.